README.md

```diff
@@ -35,14 +35,42 @@ Rails tests:
 
 Running React/Javascript unit tests:
 
-`yarn test`
+`yarn unit`
 
 Running React/Javascript e2e tests:
 
+In order to run e2e tests you have to:
+* set 2 environment variables to login into Saleforce.
+* run your server locally.
+
+**env variables**
+
+You have to set it in your environment and not in your .env file since it's being used only in Ruby and e2e runs in Javascript.
+
+Set:
+
+```
+E2E_SALESFORCE_USERNAME=dahlia-leasing-agent@exygy.com.full
+E2E_SALESFORCE_PASSWORD=<ask the team for this password>
+```
+
+**run server**
+
+Run your Rails server locally in port 3000:
+
+`bundle exec rails server -p 3000`
+
+to run tests:
+
```

Comment:
is there a way to run them in a non-headless way?

Reply:
you can do it from the code. Change the `headless` attribute to false. 
https://github.com/Exygy/sf-dahlia-lap/pull/90/files#diff-e421025dd63b8713f868239494afad38R13

Reply:
can you add that to the readme? or is there an easy way to make it configurable from with e.g. a flag from the command line?

Reply:
It was in the readme. I improved it a bit:
https://github.com/Exygy/sf-dahlia-lap/tree/chores/init-e2e-test-%23159808509

Reply:
I don't see it int he readme still but maybe I'm just blind?